### PR TITLE
Added name to the list 

### PR DESCRIPTION
**Pull request by dominikkawka**
@laurafitzgerald

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ Laura makes some changes
 
 made some other changes
 
-
```
Comment (laurafitzgerald) [inline README.md:7]: @dominikkawka why was this line removed?

File: README.md [hunk 2]
```diff
@@ -4,7 +4,8 @@ Laura makes some changes
...
 sample branch changes
 
-Add your name and and one other line here:
\ No newline at end of file
+Add your name and and one other line here:
+
+Dominik
```
Comment (laurafitzgerald) [inline README.md]: @dominikkawka could you add another line here with a quote or something similar

**Review by laurafitzgerald — CHANGES_REQUESTED** [review]
left some comments in line

File: README.md (hunk 1 above)
Comment (dominikkawka) [reply README.md:7]: Because there was only 1 space in the other sentences

**Review by laurafitzgerald — APPROVED** [review]
